src/cli.c

```diff
@@ -385,17 +392,36 @@ static int get_password_verify(char *prompt, char *password, int count)
 static int import_keys(user_options_t *options)
 {
     int status = 0;
-    char *host = options->host ? strdup(options->host) : NULL;
-    char *user = options->user ? strdup(options->user) : NULL;
-    char *pass = options->pass ? strdup(options->pass) : NULL;
-    char *key = options->key ? strdup(options->key) : NULL;
-    char *mnemonic = options->mnemonic ? strdup(options->mnemonic): NULL;
-    char *mnemonic_input = NULL;
+    char *host = options->host ? strdup(options->host) : "";
+    char *apikey = options->apikey ?
+        strdup(options->apikey) : calloc(BUFSIZ, sizeof(char));
+    char *enc_access_str = options->enc_access ?
+        strdup(options->enc_access): NULL;
+    char *user_passphrase = options->user_passphrase ?
+            strdup(options->user_passphrase) : NULL;
     char *user_file = NULL;
     char *root_dir = NULL;
+    char *new_enc_passphrase = calloc(BUFSIZ, sizeof(char));
+    char *user_input = calloc(BUFSIZ, sizeof(char));
     int num_chars;
 
-    char *user_input = calloc(BUFSIZ, sizeof(char));
+    printf("import_keys host: %s\n", host);
+
+    if (!apikey) {
+        apikey = calloc(BUFSIZ, sizeof(char));
+        printf("Project API key: ");
+        get_input(apikey);
+        printf("\n");
+        num_chars = strlen(apikey);
+
+        if (num_chars == 0) {
+            // TODO: add link
+            printf("Project API key required. See <link here>\n");
```

Comment:
What should we link here?

Reply:
Docs explaining where to get an apikey or something was my thought.